core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.ts

```diff
@@ -205,12 +205,20 @@ angular.module('oppia').directive('skillsList', [
             return (skillSerialNumber + 1);
           };
 
+          ctrl.toggleEditOptions = function(skillId) {
+            ctrl.selectedIndex = ctrl.selectedIndex ? null : skillId;
```

Comment:
I think 'toggle' is generally used to indicate that there are only two options to choose from. Can we rename this to avoid confusion?

Reply:
Fixed.